README.nob.md

```diff
@@ -0,0 +1,16 @@
+# Lavinia-client
+Denne guiden på andre språk:
+* [English](README.md)
+
+# Forutsetninger
+Du har installert:
+* [Node.js](https://nodejs.org)
+* [Yarn](https://yarnpkg.com)
+
+# For å kjøre utviklingsbygget
+1. Klon repositoriet ned til din foretrukne mappe
+2. Åpne ditt favorittskall og naviger til den tidligere nevnte mappen
```

Comment:
I think we should call it kommandovindu or terminal, as people are most likely to use a CLI to execute these commands.